### PR TITLE
add issues and pr templates

### PR DESCRIPTION
**Pull request by inkhey**
Add github templates to make life a bit easier for : 
- maintainer
- external contributors

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+### Version information
+
+<!-- Optional - describe the environment which reveals the bug: -->
+
+- Version: [e.g. 0.22]
+- Device / System: [e.g. Debian Buster]
+
+## Diagnostic
+
+<!-- Explain what may cause the bug. -->
```
Comment (raphj) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]:
```suggestion
<!-- Optional - explain what may cause the bug. -->
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+<!-- If you know which component may be affected, feel free to mention it. -->
+
+### How to reproduce
+
+<!-- Detail steps to reproduce the bug: -->
+
+
+### Version information
+
+<!-- Optional - describe the environment which reveals the bug: -->
```
Comment (raphj) [inline .github/ISSUE_TEMPLATE/bug_report.md:23]: I don't think this should be optional. Weird behavior on Preview Generator are highly dependent of the environment, and I find myself asking people for more information when they are not given

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Using  Windows/MacOSX/Non-Debian Linux Distrib ?
```
Comment (raphj) [inline .github/ISSUE_TEMPLATE/config.yml:3]:
```suggestion
  - name: Using Windows / MacOS / a non-Debian Linux Distribution?
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+    url: https://github.com/algoo/preview-generator/issues/241
+    about: Metaticket about support for non-debian/ubuntu distrib.
```
Comment (raphj) [inline .github/ISSUE_TEMPLATE/config.yml:5]:
```suggestion
    about: Metaticket about support for non-Debian/Ubuntu distributions.
```

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<!-- Put here the original issue link if exist-->
```
Comment (raphj) [inline .github/pull_request_template.md:1]:
```suggestion
<!-- Put here the link to the original issue if there is one -->
```

**Review by raphj — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (inkhey) [reply .github/ISSUE_TEMPLATE/bug_report.md:23]:
```suggestion
<!-- describe the environment which reveals the bug: -->
```

**Review by raphj — APPROVED** [review]

**Pull request merged**